osu.Game.Rulesets.Taiko/Mods/TaikoModSimplified.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Localisation;
+using osu.Game.Beatmaps;
+using osu.Game.Beatmaps.ControlPoints;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Taiko.Beatmaps;
+using osu.Game.Rulesets.Taiko.Objects;
+
+namespace osu.Game.Rulesets.Taiko.Mods
+{
+    public class TaikoModSimplified : Mod, IApplicableToBeatmap
+    {
+        public override string Name => "Simplified";
```

Comment:
Name feels uninspired. Alternative suggestions would be something with "quarters". Or maybe "Four On The Floor" or something (to annotate the worst you have to deal with is 1/4s).

Reply:
thoughts on "Quarterize"?